packages/salesforcedx-vscode-apex/src/views/testOutlineProvider.ts

```diff
@@ -37,91 +41,72 @@ export class ApexTestOutlineProvider implements vscode.TreeDataProvider<TestNode
   public onDidChangeTreeData = this.onDidChangeTestData.event;
 
   private apexTestMap: Map<string, TestNode> = new Map<string, TestNode>();
-  private rootNode: TestNode | null;
-  public testStrings: Set<string> = new Set<string>();
```

Comment:
not necessary, you can derive this from `apexTestInfo` if/when the one command that uses it (runAllTests) gets called